data.sql

```diff
@@ -38,3 +42,36 @@ update animals set owner_id=4 where name = 'Squirtle';
 update animals set owner_id=4 where name = 'Blossom';
 update animals set owner_id=5 where name = 'Angemon';
 update animals set owner_id=5 where name = 'Boarmon';
+
+--PROJECT 4
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, 'Apr 23, 2000');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, 'Jan 17, 2019');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, 'May 04, 1981');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, 'Jun 08, 2008');
+
+INSERT INTO specializations (species_id, vets_id) VALUES (1, 1);
+INSERT INTO specializations (species_id, vets_id) VALUES (1, 3);
+INSERT INTO specializations (species_id, vets_id) VALUES (2, 3);
+INSERT INTO specializations (species_id, vets_id) VALUES (2, 4);
+
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 1, 'May 24, 2020');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 3, 'Jun 22, 2020');
```

Comment:
- [ ] Kindly make sure to write the correct date, it's July, not June for the month as per the requirements.

Reply:
Thank you, I've missed there. Now it's fixed. No changes in queries results.